controllers/managedfusion_controller.go

```diff
@@ -294,6 +305,7 @@ func (r *ManagedFusionReconciler) initReconciler(ctx context.Context, req ctrl.R
 
 	r.smtpConfigData = &smtpConfig{}
 	r.pagerDutyConfigData = &pagerDutyConfig{}
+	r.remoteWriteConfig = &remoteWriteConfig{}
```

Comment:
This is an optional section. As such we should not create an instance for the struct until we see a need for it. 
This will make sure that if the section is not present the value of `r.remoteWriteConfig` will stay `nil`